### PR TITLE
Build updating fixes

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
Ready/In Progress/In Hold(Reason for hold)

## Description
Only perform configure and update on non-nightly master builds. Reconfigure client before disabling instances since authorization has probably expired.

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

File: Tests/configure_and_test_integration_instances.py [hunk 1]
```diff
@@ -612,6 +612,8 @@ def main():
         if not success:
             postupdate_fails.add((instance_name, integration_of_instance))
 
+    # reinitiate the client since its authorization has probably expired by now
+    client = demisto_client.configure(base_url=server, username=username, password=password, verify_ssl=False)
```
Comment (yaakovi) [inline Tests/configure_and_test_integration_instances.py]: switch to api key instead of user-password combo.
do it in another PR

File: .circleci/config.yml [hunk 2]
```diff
@@ -173,7 +173,9 @@ jobs:
             demisto-sdk create -a $CIRCLE_ARTIFACTS
             # create zip with contents of content_new.zip and content_test.zip for use in updating content on instances
             cp "$CIRCLE_ARTIFACTS/content_new.zip" "$CIRCLE_ARTIFACTS/all_content.zip"
-            zip -j "$CIRCLE_ARTIFACTS/all_content.zip" "$CIRCLE_ARTIFACTS/bundle_test/*"
+            unzip -q "$CIRCLE_ARTIFACTS/content_test.zip" -d "$CIRCLE_ARTIFACTS/test_content"
```
Comment (yaakovi) [inline .circleci/config.yml]: change the destination of the temp folder (use CIRCLE_ARTIFACTS only for actual artifacts).
do it in another PR.

Comment (avidan-H) [reply .circleci/config.yml]: what does it matter if I’m deleting it in the same build step anyway?

Comment (yaakovi) [reply .circleci/config.yml]: if step crashed before it removed the folder, it will be posted as an artifact.
don't know what are the effects for so many files as artifacts and i don't want to find out...

Comment (avidan-H) [reply .circleci/config.yml]: legit. It shall be as you say sensei. 
image
